### DISCUSSION TITLE
Add Games to Steam on Windows?

### DISCUSSION THREAD

**Discussion by H-Tevin** [Heroic-Games-Launcher/HeroicGamesLauncher#1531] [Q&A] [answered]
I am trying to figure out how to add games on Heroic Launcher to Steam.
The wiki only describes how to add games with the Heroic Bash Launcher, but that is only for Linux.
[Other guides](https://steamsplay.com/how-to-add-games-from-epic-to-steam-guide/) had instructions for adding Epic Games to Steam by copying the URL of the EGS shortcut. These worked to launch EGS games through Steam, but when I tried a similar thing with Heroic, the Target of the shortcut is greyed out and I am unable to copy it.

Is there a way to add Heroic Games to Steam? Preferably using the Bash Launcher, since that seems to take care of everything, but even manually would be great.
Thanks for your help!

**Comment by redromnon** [comment, marked as answer]
There's a PR currently being worked on for adding shorcuts to Steam. This includes adding to Windows too.

**Reply by H-Tevin** [reply]
Thank you! I look forward to seeing it implemented
